quinn/schema_helpers.py

```diff
@@ -0,0 +1,72 @@
+from pyspark.sql import types as T
+
+
+def print_schema_as_code(dtype: T.DataType) -> str:
```

Comment:
I actually thought the argument here was going to be `T.StructType`.  It looks like that's what you're using in the test and everything is working fine...  Can you please educate me?